client/state/sharing/services/selectors.js

```diff
@@ -92,6 +93,15 @@ export function getEligibleKeyringServices( state, siteId, type ) {
 			return false;
 		}
 
+		// Omit if service is settings-oriented and user cannot manage
+		if (
+			'google_my_business' === service.ID &&
+			( ! canCurrentUser( state, siteId, 'manage_options' ) ||
+				! config.isEnabled( 'google-my-business' ) )
```

Comment:
You removed `config.isEnabled( 'google-my-business' )` elsewhere, but not here, why?

Reply:
Because M1 is deployed and `google-my-business` is enabled on all environments (except `desktop` I think).
Thus I'm disabling `google-my-business` in stage and production making it a feature wall for M2.